### ISSUE TITLE
how to add i18n.locale: zh-CN to kibana

### ISSUE THREAD

**Issue by Pandoraemon** [elastic/cloud-on-k8s#1182]
how to add i18n.locale: zh-CN to kibana,  looks not work with 
apiVersion: kibana.k8s.elastic.co/v1alpha1
kind: Kibana
metadata:
  name: quickstart
spec:
  version: 7.1.0
  image:  kibana:7.1.0
  nodeCount: 1
  featureFlags:
      i18n.locale: "zh-CN"
  elasticsearchRef:
    name: quickstart
  http:
    service:
      spec:
        type: NodePort
  volumeClaimTemplates:
    - metadata:
        name: kibana-data
      spec:
        accessModes:
        - ReadWriteOnce
        resources:
          requests:
            storage: 10Gi
        storageClassName: fast-disks

**Comment by sebgl** [comment]
Hi @Pandoraemon,
This is currently not possible in ECK v0.8, since it does not support custom Kibana configuration.

However it will be possible in ECK v0.9, through the `config` section of the Kibana resource:

```yaml
apiVersion: kibana.k8s.elastic.co/v1alpha1
kind: Kibana
metadata:
  name: kibana-sample
spec:
  version: "7.2.0"
  config:
    i18n.locale: zh-CN
  nodeCount: 1
  elasticsearchRef:
    name: "elasticsearch-sample"
```

**Comment by sebgl** [comment]
Closing this now since fixed in Master, will be released with 0.9.
